closed/autoconf/custom-hook.m4

```diff
@@ -667,17 +670,21 @@ AC_DEFUN([OPENJ9_GENERATE_TOOL_WRAPPERS],
 [
   MSVC_BIN_DIR=$($DIRNAME $CC)
   SDK_BIN_DIR=$($DIRNAME $RC)
+  FIXPATH2="$TOPDIR/closed/fixpath2.sh"
 
   mkdir -p "$OPENJ9_TOOL_DIR"
-  OPENJ9_GENERATE_TOOL_WRAPPER([cl], [$CC])
-  OPENJ9_GENERATE_TOOL_WRAPPER([lib], [$AR])
-  OPENJ9_GENERATE_TOOL_WRAPPER([link], [$LD])
-  OPENJ9_GENERATE_TOOL_WRAPPER([ml], [$MSVC_BIN_DIR/ml])
-  OPENJ9_GENERATE_TOOL_WRAPPER([ml64], [$MSVC_BIN_DIR/ml64])
-  OPENJ9_GENERATE_TOOL_WRAPPER([rc], [$RC])
-  OPENJ9_GENERATE_TOOL_WRAPPER([mc], [$SDK_BIN_DIR/mc])
-  OPENJ9_GENERATE_TOOL_WRAPPER([nasm], [$NASM])
-  OPENJ9_GENERATE_TOOL_WRAPPER([java], [$JAVA])
-  OPENJ9_GENERATE_TOOL_WRAPPER([jar], [$JAR])
-  OPENJ9_GENERATE_TOOL_WRAPPER([javac], [$JAVAC])
+  OPENJ9_GENERATE_TOOL_WRAPPER([cl], [$FIXPATH2], [$CC])
+  OPENJ9_GENERATE_TOOL_WRAPPER([lib], [$FIXPATH2], [$AR])
+  OPENJ9_GENERATE_TOOL_WRAPPER([link], [$FIXPATH2], [$LD])
+  OPENJ9_GENERATE_TOOL_WRAPPER([ml], [$FIXPATH2], [$MSVC_BIN_DIR/ml])
+  OPENJ9_GENERATE_TOOL_WRAPPER([ml64], [$FIXPATH2], [$MSVC_BIN_DIR/ml64])
+  OPENJ9_GENERATE_TOOL_WRAPPER([rc], [$FIXPATH2], [$RC])
+  OPENJ9_GENERATE_TOOL_WRAPPER([mc], [$FIXPATH2], [$SDK_BIN_DIR/mc])
+  OPENJ9_GENERATE_TOOL_WRAPPER([nasm], [$FIXPATH2], [$NASM])
+  OPENJ9_GENERATE_TOOL_WRAPPER([clang], [$FIXPATH2], [$OPENJ9_CLANG])
+
+  # fixpath2 can't handle classpaths, or @<file_name> arguments
+  OPENJ9_GENERATE_TOOL_WRAPPER([java], [$FIXPATH], [$JAVA])
+  OPENJ9_GENERATE_TOOL_WRAPPER([jar], [$FIXPATH], [$JAR])
+  OPENJ9_GENERATE_TOOL_WRAPPER([javac], [$FIXPATH],[$JAVAC])
```

Comment:
Since you're changing all these lines, could you please sort them in each group by command name.